packages/terra-embedded-component/src/EmbeddedComponent.jsx

```diff
@@ -1,31 +1,146 @@
+/* global SystemJS */
 import React from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
-import styles from './EmbeddedComponent.scss';
+import axios from 'axios';
+import 'systemjs/dist/system';
+import { loadStylesheet } from './assets';
 
-const cx = classNames.bind(styles);
+// Expose AMD required globals on the window
+window.define = window.SystemJS.amdDefine;
+window.require = window.requirejs = SystemJS.amdRequire;
+
+const DefaultTimeout = () => <p>Component mounting is timed out.</p>;
```

Comment:
Don't do this.... The component should throw an error if it times out and it should be the responsibility of the parent component to catch that.